extensions/lifecycle/tasks/LifecycleUpdateExpirationTask.js

```diff
@@ -0,0 +1,125 @@
+const async = require('async');
+
+const errors = require('arsenal');
+const ObjectMD = require('arsenal').models.ObjectMD;
+const BackbeatTask = require('../../../lib/tasks/BackbeatTask');
+const ActionQueueEntry = require('../../../lib/models/ActionQueueEntry');
+
+class LifecycleUpdateExpirationTask extends BackbeatTask {
+    /**
+     * Process a lifecycle object entry
+     *
+     * @constructor
+     * @param {LifecycleObjectProcessor} proc - object processor instance
+     */
+    constructor(proc) {
+        const procState = proc.getStateVars();
+        super();
+        Object.assign(this, procState);
+    }
+
+    _getMetadata(entry, log, done) {
+        const { bucket, key, version } = entry.getAttribute('target');
+        this.backbeatClient.getMetadata({
```

Comment:
there should be a call here to get an instance of the client. ex. https://github.com/scality/backbeat/blob/development/8.4/extensions/lifecycle/tasks/LifecycleUpdateTransitionTask.js#L25-L26